packages/cli/src/databases/migrations/common/1693491613982-ExecutionSoftDelete.ts

```diff
@@ -0,0 +1,19 @@
+import type { MigrationContext, ReversibleMigration } from '@/databases/types';
+
+/**
+ * Add an indexed column `deletedAt` to track soft-deleted executions.
+ * Add an index on `stoppedAt`, used by executions pruning.
+ */
+export class ExecutionSoftDelete1693491613982 implements ReversibleMigration {
+	async up({ schemaBuilder: { addColumns, column, createIndex } }: MigrationContext) {
+		await addColumns('execution_entity', [column('deletedAt').timestamp()]);
```

Comment:
I was adding `createdAt` here because we are currently using `startedAt` incorrectly as `createdAt` in queue mode, and I wanted to avoid creating 2 separate migrations.
But now that the execution data is in a separate table, perhaps another migration isn't actually going to be that bad. if that's the case, we should also remove the field from the entity.